api/v1beta2/network_types.go

```diff
@@ -91,26 +143,36 @@ type ClassicELB struct {
 	// SecurityGroupIDs is an array of security groups assigned to the load balancer.
 	SecurityGroupIDs []string `json:"securityGroupIds,omitempty"`
 
-	// Listeners is an array of classic elb listeners associated with the load balancer. There must be at least one.
-	Listeners []ClassicELBListener `json:"listeners,omitempty"`
+	// ClassicELBListeners is an array of classic elb listeners associated with the load balancer. There must be at least one.
+	ClassicELBListeners []ClassicELBListener `json:"listeners,omitempty"`
 
 	// HealthCheck is the classic elb health check associated with the load balancer.
 	HealthCheck *ClassicELBHealthCheck `json:"healthChecks,omitempty"`
 
-	// Attributes defines extra attributes associated with the load balancer.
-	Attributes ClassicELBAttributes `json:"attributes,omitempty"`
+	// ClassicElbAttributes defines extra attributes associated with the load balancer.
+	ClassicElbAttributes ClassicELBAttributes `json:"attributes,omitempty"`
 
 	// Tags is a map of tags associated with the load balancer.
 	Tags map[string]string `json:"tags,omitempty"`
+
+	// V2Listeners is an array of listeners associated with the load balancer. There must be at least one.
+	V2Listeners []Listener `json:"v2Listeners,omitempty"`
```

Comment:
Good question! I only exposed it because the other one was exposed as well. Do you think it's usually not needed at all? I'm happy not to include this at all. It doesn't seem like users would define extra listeners.

That said, there might be an option to add an extra health check or something... 🤔

Reply:
In any case, I renamed it. :)